notebooks/labs37_notebooks/SentimentAnalysisFor_TwitterBot.ipynb

```diff
@@ -349,7 +1947,8 @@
       "source": [
         "# Load Pretrained Word2Vec\n",
         "#embed = hub.load(\"https://tfhub.dev/google/Wiki-words-250/2\")\n",
-        "embed = hub.load(\"https://tfhub.dev/google/Wiki-words-500/2\")"
+        "embed = hub.load(\"https://tfhub.dev/google/Wiki-words-500/2\")\n",
+        "#embed = GloVe(name='6B', dim=100)"
```

Comment:
If not using GloVe can we delete line 1951 and maybe just include a note about trying GloVe in the documentation we're leaving for the next cohort?